far/FarEng.hlf.m4

```diff
@@ -2750,6 +2750,8 @@ $ #Viewer: control keys#
  In #hex# and #dump# ~view modes~@ViewerMode@, #Ctrl+Left# and
 #Ctrl+Right# keys shift the content within the window one byte at a time
 in the corresponding direction.
+ In #hex# ~view modes~@ViewerMode@, #Alt+Left# and
+#Alt+Right# keys remove or add, respectively.
```

Comment:
"... keys remove or add" what? One byte worth of width, or 16 bytes, or what?

Also, should be `~view mode~` in singular.

Also, please consider removing the hyperlink altogether. It is already displayed in the previous paragraph, not much point repeating it here.

Also, please add an empty line above the inserted text to separate paragraphs.

Also, please consider updating the `@ViewerMode` page accordingly.

Reply:
Modifications were done. Some typos may applie as I'm a non english nor russian native. Also, in the russian language file, I wrote the same english text as on the other files.

Reply:
Thank you. Never mind typos.